packages/sdk-react/src/components/counterparts/CounterpartDetails/CounterpartDetails.test.tsx

```diff
@@ -638,4 +652,30 @@ describe('CounterpartDetails', () => {
       });
     });
   });
+
+  describe('# Make default contact for counterpart', () => {
+    const setupTest = (organizationEmail: string, contactEmail: string) => {
+      const contacts = [{ email: contactEmail, is_default: true }];
+      const organization = {
+        email: organizationEmail,
+        is_customer: true,
+        is_vendor: false,
+        legal_name: 'Test Corp',
+        phone: '123-456-7890',
+      };
+      return prepareCounterpartOrganization(organization, {}, contacts);
+    };
+
+    describe('prepareCounterpartOrganization', () => {
+      it('should return isEmailDefault as true when contact email matches organization email', () => {
```

Comment:
For now, we have enough of these tests to deploy the feature. After PR merged, please don't forget to create an issue so that we can add tests of actual actions. And let's schedule a meeting, so I can talk about the nuances without rushing.